EventListener/ChannelSubscriber.php

```diff
@@ -55,12 +55,8 @@ public function onAddChannel(ChannelEvent $event)
         $integration = $this->helper->getIntegrationObject('DoNotContactExtras');
         if ($integration && $integration->getIntegrationSettings()->isPublished()) {
             $event->addChannel(
-                'phone',
-                [LeadModel::CHANNEL_FEATURE   => []]
-            );
-            $event->addChannel(
-                'sms',
-                [LeadModel::CHANNEL_FEATURE   => []]
+                'Phone / SMS',
+                [LeadModel::CHANNEL_FEATURE => []]
```

Comment:
Does case matter?

Reply:
no. channels are just an array of strings.